Formula/fpc.rb

```diff
@@ -51,6 +74,18 @@ def install
 
     # Generate a default fpc.cfg to set up unit search paths
     system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{prefix}/etc/fpc.cfg"
+
+    if OS.linux?
+      # On Linux, non-executable IDE support files get built and end up in bin.
+      # Move them somewhere else instead.
+      mkdir_p pkgshare/"ide"
+      mv Dir[bin/"*.{ans,tdf,pt}"], pkgshare/"ide"
```

Comment:
```suggestion
      (pkgshare/"ide").install Dir[bin/"*.{ans,tdf,pt}"]
```
or even

```suggestion
      (pkgshare/"ide").install bin.glob("*.{ans,tdf,pt}")
```

Reply:
I will stick with the former (not replacing `Dir[...]` with `bin.glob`) since it's more consistent with what's in core right now. A quick grep shows a lot of places where we could substitute `lib.glob`, `libexec.glob`, `man7.glob`, etc., if that's a desired style direction we want to move towards.

Reply:
They're not always substitutes for each other: `Dir["*"]` returns `String`s, while `Pathname#glob` returns `Pathname`s.

Reply:
Right, I was looking specifically at usages of `Dir#glob` where the first path element is `lib`, `share`, etc. Examples in formulae include `python@3.{7,8,9}`, `gcc`, `ghc`, `gdal`, `wrangler`, `xctool`.

Reply:
Ah yes, I see. I don't really have a preference -- each form has its uses.

I'm inclined to say that we should use `Pathname#glob` whenever we have a `Pathname` and `Dir[]` whenever we don't, but that would require somewhat meaningless changes in a bunch of places.